### ISSUE TITLE
GHGI_v2_Express_Ext inventory not properly read in CH4 simulations

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#2036]
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard / GCST

### Description of your issue or question

@djvaron found the GHGI emissions (express extension option) were not properly read in GEOS-Chem CH4 simulations for 14.2.0 through 14.2.2. The issue was the use of logicals in HEMCO_Config.rc. Currently the GHGI express extension option is bracketed with:

```
(((GHGI_v2_Express_Ext.and..not.GHGI_v2
```

This is to make sure that if users have both GHGI_v2 and GHGI_v2_Express_Ext set to true that only the official GHGI_v2 product will be used to avoid double counting. However, it turns out HEMCO can only handle use of `.or.` and `.not.` within the logical statements -- `.and.` is not supported. To get around this, we can separate the statement into two lines:

```
(((GHGI_v2_Express_Ext
(((.not.GHGI_v2
```

A fix for this will be included in 14.2.3.

**Comment by yantosca** [comment]
Closed by PR #2043

**Comment by yantosca** [comment]
We can now close this issue since PR #2042 has been merged into the `dev/no-diff-to-benchmark` branch.
